pipeline/packer/jenkins-worker.json

```diff
@@ -0,0 +1,35 @@
+{
+  "_description": "This template produces a Jenkins worker.",
+ "variables": {
+    "repository": "{{env `DOCKER_REPOSITORY`}}",
+    "tag": "{{env `BUILD_TAG`}}",
+    "docker_registry_url": "{{env `DOCKER_REGISTRY_URL`}}"
+  },
+  "builders": [
+    {
+      "type": "docker",
+      "image": "temporary/jenkins-worker:latest",
+      "pull": false,
```

Comment:
Note that this line is here to make sure we get the base image from the local Docker repository and don't try to pull from upstream (ie Docker hub)